lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -6844,6 +6958,60 @@ def air_loop_hvac_supply_air_temperature_reset_type(air_loop_hvac)
     return 'warmest_zone'
   end
 
+  # Calculate the wwr of a surface
+  #
+  # @param surface [OpenStudio::Model::Surface]
+  # @return [Float] window to wall ratio of a surface
+  def get_wwr_of_a_surface(surface)
+    surface_area = surface.grossArea
+    surface_fene_area = 0.0
+    surface.subSurfaces.sort.each do |ss|
+      next unless ss.subSurfaceType == 'FixedWindow' || ss.subSurfaceType == 'OperableWindow' || ss.subSurfaceType == 'GlassDoor'
+
+      surface_fene_area += ss.netArea
+    end
+    return surface_fene_area / surface_area
+  end
```

Comment:
This should probably be moved to `Standards.Surface.rb` and perhaps be renamed `surface_get_wwr` or something similar to be consistent with other surface related methods.

Same for `get_door_ratio_of_a_surface`.